scripts/manual_validation_results.qmd

```
@@ -0,0 +1,1327 @@
+---
+title: "Results manual validation"
+format: html
+editor: visual
+---
+
+# Dataset and packages
+
+```{r message = FALSE, warning = FALSE}
+
+final_data <- read.csv(here::here("data", "manual_validation_processed.csv"), sep = ",")
```

please make sure that the dataset you're reading in does not contain any whitespaces, as otherwise you might get silent errors. See this issue in a few columns right now and relevant comments in the `manual_validation_data_processing.R` script). I think this affects the `trn1`, `trn2`, `recruitment_status_reg1`, and `recruitment_status_reg2` columns, and possibly more

---

This should be solved now with the newest processing script